README.md

```diff
@@ -491,26 +491,78 @@ If you specify both **alert** and **data**, target device will receive both noti
+`mongoose_push_notification_send_time_microsecond_sum{error_category=${CATEGORY},error_reason=${REASON},service=${SERVICE},status=${STATUS}}`
+`mongoose_push_notification_send_time_microsecond_count{error_category=${CATEGORY},error_reason=${REASON},service=${SERVICE},status=${STATUS}}`
 
-* `mongoose_push_apns_state_get_default_topic_count`
-* `mongoose_push_notification_send_time_bucket{error_category=${CATEGORY},error_reason=${REASON},service=${SERVICE},status=${STATUS},le=${LENGTH}}`
-* `mongoose_push_notification_send_time_sum{error_category=${CATEGORY},error_reason=${REASON},service=${SERVICE},status=${STATUS}}`
-* `mongoose_push_notification_send_time_count{error_category=${CATEGORY},error_reason=${REASON},service=${SERVICE},status="${STATUS}}`
 Where:
-* **CATEGORY** is an arbitrary error category term or empty string
-* **REASON** is an arbitrary error reason term or empty string
-* **SERVICE** is either `fcm` or `apns`
-* **STATUS** is either `success` or `error`
-* **LENGTH** is either `100` or `250` or `500` or `1000` or `+Inf`
+* `STATUS` is `"success"` for the successful notifications or `"error"` in all other cases
+* `SERVICE` is either `"apns"` or `"fcm"`
+* `CATEGORY` is an arbitrary error category term (in case of `status="error"`) or an empty string (when `status="success"`)
+* `REASON` is an arbitrary error reason term (in case of `status="error"`) or an empty string (when `status="success"`)
+* `LE` defines the `less or equal` values for buckets, currently `1000`, `10_000`, `25_000`, `50_000`, `100_000`, `250_000`, `500_000`, `1000_000` or `+Inf`
+
+> **NOTE**
+> A mesurment of value 50_000 will be added to all buckets which are less or equal to 50_000.
+> In this case these are buckets `50_000`, `100_000`, `250_000`, `500_000`, `1000_000` and `+Inf`
+
+This histogram metric shows the distribution of times needed to:
+1. Select a worker (this may include waiting time when all workers are busy).
+2. Send a request.
+3. Get a response from push notifications provider.
+
+##### Counters
+
+* `mongoose_push_supervisor_init_count{service=${SERVICE}}` - Counts number of push notification service supervisor starts.
+  The `SERVICE` variable can take `"apns"` or `"fcm"` as a value.
+  This metrics is update when MongoosePush starts or later when the underlying supervision tree is terminated and the error is propagate to the main application supervisor.
+* `mongoose_push_apns_state_init_count` - Counts number of APNS state initialisations.
+* `mongoose_push_apns_state_terminate_count` - Counts number of APNS state terminations.
+* `mongoose_push_apns_state_get_default_topic_count` - Counts number of default topic reads from cache.
+
+#### How to quickly see all metrics
+
+```bash
+curl -k https://127.0.0.1:8443/metrics
+```
+
+The above command assumes that MongoosePush runs on `localhost` and listens on port `8443`.
+Please, mind the `HTTPS` protocol, metrics are hosted on the same port as other API.
```

Comment:
```suggestion
Please, mind the `HTTPS` protocol, metrics are hosted on the same port as other API endpoints.
```
How about this as compromise? 
`the same as` seems more grammatically correct than `the same than`